docs/src/extensions.md

```diff
@@ -97,3 +176,5 @@ Similar to the proc-ps implementation and unlike GNU/Coreutils, `uptime` provide
 ## `base32/base64/basenc`
 
 Just like on macOS, `base32/base64/basenc` provides `-D` to decode data.
+
+[^1] https://en.cppreference.com/w/c/io/fprintf
```

Comment:
I don't think this is the best source for that. Generally, we should refer to GNU docs or the POSIX specification.

You might find the info on one of these pages:
- https://pubs.opengroup.org/onlinepubs/9699919799/utilities/printf.html
- https://pubs.opengroup.org/onlinepubs/9699919799/functions/strtod.html

Reply:
Thanks! I think this has what we need: https://pubs.opengroup.org/onlinepubs/9699919799/basedefs/V1_chap05.html#tag_05

(GNU docs reword the same thing, and I think C99 is also very similar in wording)